scripts/qa_crowbarsetup.sh

```diff
@@ -3932,6 +3932,20 @@ function onadmin_cloudupgrade_reboot_and_redeploy_clients()
     # TODO: restart any suspended instance?
 }
 
+function onadmin_prepare_crowbar_upgrade()
+{
+    if iscloudver 4minus ; then
+        complain 11 "This upgrade path is only supported for Cloud 5+"
+    else
+        # using the API, due to missing crowbar cli integration
+        local digest="--digest -u crowbar:crowbar"
+        # move nodes to upgrade mode
+        safely curl -X POST $digest $crowbar_api/installer/upgrade/prepare
+        # stopping services
+        safely curl -X POST $digest $crowbar_api/installer/upgrade/services
```

Comment:
like this we unfortunately have no idea when the server is done with finishing the request...